refiner/app/api/v1/configurations.py

```diff
@@ -1041,12 +1046,17 @@ async def run_configuration_test(
 
     # call the testing service
     # business logic around **how** inline testing works is in services/testing.py
-    result = await inline_testing(
-        xml_files=original_xml_files,
-        configuration=configuration,
-        primary_condition=primary_condition,
-        jurisdiction_id=jd,
-    )
+    try:
+        result = await inline_testing(
+            xml_files=original_xml_files,
+            configuration=configuration,
+            primary_condition=primary_condition,
+            jurisdiction_id=jd,
+        )
+    except XMLValidationError:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=XML_FILE_ERROR
```

Comment:
not sure if I'm crossing abstraction layer metaphors here, but added this except to give the user a non "500 Server Error" message in the instance where something goes wrong in the parsing process for testing